### DISCUSSION TITLE
Add item to Backpex.ItemAction/1

### DISCUSSION THREAD

**Discussion by pehbehbeh** [naymspace/backpex#484] [Ideas]
If we would pass the item to the [`icon/1` function](https://hexdocs.pm/backpex/0.5.1/Backpex.ItemAction.html#c:icon/1), we could customise the icon based on the content (e.g. add a badge).

Specific use case: We have an icon that leads to a nested resource (events.subtitles). A badge with the number of subtitles could be displayed on the icon.

**Comment by SebastianMueller87** [comment]
It got implemented in [0.7.0](https://github.com/naymspace/backpex/releases/tag/0.7.0) via [#592](https://github.com/naymspace/backpex/pull/592)
